clinica/pydra/statistics_volume/pipeline.py

```diff
@@ -0,0 +1,238 @@
+import pydra
+from nipype.algorithms.misc import Gunzip
+from pydra import Workflow
+from pydra.tasks.nipype1.utils import Nipype1Task
+
+from clinica.pydra.engine import clinica_io
+
+
+def _check_pipeline_parameters(parameters: dict) -> dict:
+    """Check the parameters passed to the pipeline.
+
+    Parameters
+    ----------
+    parameters : dict
+        Dictionary of parameters to analyze.
+
+    Returns
+    -------
+    dict :
+        Cleaned dictionary of parameters.
+    """
+    from clinica.utils.exceptions import ClinicaException
+
+    # PET-Volume pipeline
+    if parameters["orig_input_data_volume"] == "pet-volume":
+        for param in ("acq_label", "suvr_reference_region"):
+            if not parameters[param]:
+                raise ClinicaException(
+                    f"You selected pet-volume pipeline without setting --{param} flag. "
+                    "Clinica will now exit."
+                )
+
+    # Custom pipeline
+    if parameters["orig_input_data_volume"] == "custom-pipeline":
+        if not all([parameters["custom_file"], parameters["measure_label"]]):
+            raise ClinicaException(
+                "You must set --measure_label and --custom_file flags."
+            )
+
+    return parameters
+
+
+@clinica_io
+def build_core_workflow(name: str = "core", parameters={}) -> Workflow:
+    """Build the core workflow for the Statistics Volume pipeline.
+
+    Parameters
+    ----------
+    name : str, optional
+        The name of the workflow. Default="core".
+
+    parameters : dict, optional
+        Dictionary of parameters to be used
+        within the workflow.
+        Default={}.
+
+    Returns
+    -------
+    wf : Workflow
+        The core workflow.
+    """
+    from os.path import dirname, join
+    from typing import Any
+
+    import clinica.pydra.statistics_volume.task as utils
+    from clinica.utils.exceptions import ClinicaException
+    from clinica.utils.spm import spm_standalone_is_available, use_spm_standalone
+
+    if spm_standalone_is_available():
+        use_spm_standalone()
+
+    parameters = _check_pipeline_parameters(parameters)
+
+    input_name = parameters["orig_input_data_volume"]
```

Comment:
Maybe we can replace the "-" by "_" here and avoid the variable re-assignments:

```suggestion
    input_name = parameters["orig_input_data_volume"].replace("-", "_")
```